src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -741,7 +742,11 @@ internal FunctionEnvironmentReloadRequest GetFunctionEnvironmentReloadRequest(ID
         internal void SendFunctionLoadRequest(FunctionMetadata metadata, ManagedDependencyOptions managedDependencyOptions)
         {
             _functionLoadRequestResponseEvent = _metricsLogger.LatencyEvent(MetricEventNames.FunctionLoadRequestResponse);
-            _workerChannelLogger.LogDebug("Sending FunctionLoadRequest for function: '{functionName}' with functionId: '{functionId}'", metadata.Name, metadata.GetFunctionId());
+
+            if (_workerChannelLogger.IsEnabled(LogLevel.Debug))
```

Comment:
Is it a valid case where we will disable debug level log?
We have other `LogDebug` calls in the cold start path. Should all of them be fixed?

Reply:
Yes, it is valid. Even enabled by default in hosted environments, running the host in different contexts (e.g., self-hosted, custom containers) should support this.

We should do the same for other calls, but I want to use a better pattern that is both, more performant and cleaner. Will have that as a separate change, but wanted to get this one in.